cmake/cares.cmake

```diff
@@ -35,6 +35,13 @@ elseif("${gRPC_CARES_PROVIDER}" STREQUAL "package")
   if(TARGET c-ares::cares)
     set(_gRPC_CARES_LIBRARIES c-ares::cares)
     set(_gRPC_CARES_INCLUDE_DIR ${c-ares_INCLUDE_DIR})
+    if (MSVC)
+      # TODO(apolcyn): remove this hack. Our grpc/c-ares code on
```

Comment:
it looks like this change would break the distributions tests  in case the user decided to build grpc with a pre-installed version of c-ares (using a pre-installed version of our dependencies is a scenario we do want to support).